### ISSUE TITLE
fix(event): Fix event name edit

### ISSUE THREAD

**Issue by PatrickMi** [maevsi/maevsi#1030]
When a user tries to save an event with an empty name a text is shown under the apply button to indicate that there is an error with the input. When having an empty input, clicking on apply and then entering something into the name field again the prompt that tells the user that the input is wrong is coming back up.

**Comment by dargmuesli** [comment]
Do you mean "save" button when speaking about the "apply" button or another one?

**Comment by PatrickMi** [comment]
Yes. In fact this problem only occurs when trying to edit an event. This problem does not exist when a user tries this in the create event screen

**Comment by dargmuesli** [comment]
I cannot reproduce this issue, let's add this to the list of issues to look at together 👀

**Comment by dargmuesli** [comment]
Duplicate of #900
